_test_server/lib/src/handlers/test/http/custom.dart

```diff
@@ -25,8 +25,9 @@ class CustomHandler extends Handler {
 
   @override
   Future<Null> get(HttpRequest request) async {
+    final queryStatusCode = request.uri.queryParameters['status'];
```

Comment:
I didn't actually look at all the tests to see if it is ever not set, but in master we were defaulting it to a 200 OK if null, so I was restoring that logic but in a null-safe way since `int.parse` no returns non-null. More context here: https://github.com/Workiva/w_transport/pull/397#discussion_r1291580671